README.rst

```diff
@@ -214,13 +214,14 @@ Develop
 In order to experiment, play with the latest bits and develop
 improvements it is best to use a virtual environment::
 
-    mkvirtualenv tmt
+    dnf install krb5-devel libpq-devel
+    virtualenv tmt
     git clone https://github.com/psss/tmt
     cd tmt
     pip install -e .
 
 Install ``python3-virtualenvwrapper`` to easily create and enable
-virtual environments using ``mkvirtualenv`` and ``workon``. Note
+virtual environments using ``virtualenv`` and ``workon``. Note
```

Comment:
Why? This is a different command.

Reply:
Yes, I did not realize that. Thanks for clearing this on irc. I will revert.